contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are telltale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
+
+Alexey shared his advice with us on how to get your growth team winning again. Here are the key problems to investigate:
+
+## Problem 1: Your team is moving too slowly
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of [over-engineering](https://www.joelonsoftware.com/2001/04/21/dont-let-architecture-astronauts-scare-you/). "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
```

Comment:
"Engineers hate doing growth" is a little extreme.   I'm an Engineer, I love it.

More... "Many engineers don't get along with a growth working style right away"